### ISSUE TITLE
Can't connect to Presto

### ISSUE THREAD

**Issue by simple365** [rickbergfalk/sqlpad#203]
Presto-0.170
Click connection test and get error.Setting is valid.
![1](https://user-images.githubusercontent.com/2282908/28805314-b3e1ab0a-769b-11e7-96b4-278261e3eafa.png)

```
execution error:<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 404 Not Found</title>
</head>
<body><h2>HTTP ERROR 404</h2>
<p>Problem accessing /v1/execute. Reason:
<pre>    Not Found</pre></p>
</body>
</html>
```

**Comment by rickbergfalk** [comment]
Sorry for the delay in responding to this - It looks like the nodejs presto driver in use is calling a deprecated presto API. If anyone uses node and presto and would be willing to help on this it'd be much appreciated.

**Comment by rickbergfalk** [comment]
Fix for this just merged to master with #214
